pkg/doc/verifiable/credential.go

```diff
@@ -187,8 +187,8 @@ type Credential struct {
 	RefreshService *RefreshService
 }
 
-// rawCredential
-type rawCredential struct {
+// RawCredential is a basic verifiable credential
```

Comment:
I'm not excited about exporting a type name that starts with Raw. Is there a better description? Is this a BaseCredential?

Reply:
Makes sense, also thought about this afterwards my last commit...